app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PasswordsController < ApplicationController
```

Comment:
Made the decision to move password edit functionality into it's own controller to maintain RESTfulness. Can revisit this if we think should go in a different direction.

Reply:
OK I wasn't sure if the password controller was necessary, but if it works, I agree we can always revisit.